public/app/plugins/datasource/cloud-monitoring/components/GraphPeriod.tsx

```diff
@@ -27,21 +28,23 @@ export const GraphPeriod: FunctionComponent<Props> = ({ refId, onChange, graphPe
           </>
         }
       >
-        <Switch
-          data-testid={`${refId}-switch-graph-period`}
-          value={graphPeriod !== 'disabled'}
-          onChange={(e) => onChange(e.currentTarget.checked ? '' : 'disabled')}
-        />
-        <PeriodSelect
-          inputId={`${refId}-graph-period`}
-          templateVariableOptions={variableOptionGroup.options}
-          current={graphPeriod}
-          onChange={onChange}
-          selectWidth={SELECT_WIDTH}
-          disabled={graphPeriod === 'disabled'}
-          aligmentPeriods={GRAPH_PERIODS}
-        />
-      </QueryEditorRow>
-    </>
+        <HorizontalGroup>
+          <Switch
+            data-testid={`${refId}-switch-graph-period`}
+            value={graphPeriod !== 'disabled'}
+            onChange={(e) => onChange(e.currentTarget.checked ? '' : 'disabled')}
+          />
+          <PeriodSelect
+            inputId={`${refId}-graph-period`}
+            templateVariableOptions={variableOptionGroup.options}
+            current={graphPeriod}
+            onChange={onChange}
+            selectWidth={SELECT_WIDTH}
+            disabled={graphPeriod === 'disabled'}
+            aligmentPeriods={GRAPH_PERIODS}
+          />
+        </HorizontalGroup>
```

Comment:
This requires this change in the experimental package https://github.com/grafana/grafana-experimental/pull/34